actions/class.Items.php

```diff
@@ -219,6 +219,7 @@ public function editItem()
             $this->setData('isAuthoringEnabled', $hasModel);
 
             $this->setData('formTitle', __('Edit Item'));
+            
```

Comment:
If that is the only change in whole PR I would recommend just close it ( unless I'm experiencing some strong issue with github cache )